root/etc/rc.d/rc.local

```diff
@@ -5,7 +5,14 @@ exportKCL
 
 # setup root password
 if [[ ! -z "$KCL_FDI_ROOTPW" ]]; then
-  echo "root:$KCL_FDI_ROOTPW" | chpasswd >/dev/null
+  # remove possible double and/or single quotes surrounding password string
+  KCL_FDI_ROOTPW=`echo $KCL_FDI_ROOTPW | sed "s/\('\|\"\)//g"`
```

Comment:
AFAIK encrypted passwords are safe, no need to quote them at all, please remove. Explanation here: http://man7.org/linux/man-pages/man3/crypt.3.html#NOTES